pkg/agent/openflow/pipeline.go

```diff
@@ -217,6 +217,21 @@ func (c *client) podClassifierFlow(podOFPort uint32, category cookie.Category) b
 		Done()
 }
 
+func (c *client) hostNetworkForwardFlow(uplinkPort uint32, bridgeLocalPort uint32, category cookie.Category) (flows []binding.Flow) {
```

Comment:
Add some comments to explain the flows added by this func.

Reply:
Done.